mipengine/node/monetdb_interface/remote_tables.py

```diff
@@ -24,10 +24,11 @@ def get_remote_table_names(context_id: str) -> List[str]:
 )
 def create_remote_table(name: str, schema: TableSchema, monetdb_socket_address: str):
     columns_schema = convert_schema_to_sql_query_format(schema)
+    username = node_config.monetdb.username
```

Comment:
`username` and `password` should be provided as parameters, to allow for easier testing.